packages/queued-request-controller/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3]
+### Fixed
+- Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))
```

Comment:
also want to add a comment about addEthereumChain fix?

Reply:
yeah good thinking
```suggestion
- Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))
- Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))
```